feature-libs/organization/administration/components/user/form/user-form.service.ts

```diff
@@ -4,21 +4,23 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Injectable } from '@angular/core';
+import { Injectable, inject } from '@angular/core';
 import {
   UntypedFormArray,
   UntypedFormControl,
   UntypedFormGroup,
   Validators,
 } from '@angular/forms';
-import { B2BUser, B2BUserRole } from '@spartacus/core';
+import { B2BUser, B2BUserRole, FeatureConfigService } from '@spartacus/core';
 import { CustomFormValidators } from '@spartacus/storefront';
 import { FormService } from '../../shared/form/form.service';
 
 @Injectable({
   providedIn: 'root',
 })
 export class UserFormService extends FormService<B2BUser> {
+  protected readonly featureConfigService = inject(FeatureConfigService);
```

Comment:
I know that we can add `inject(Service, { optional: true })` as an option to our services. Do you know if we need to do this to make services optional?

Reply:
I belive we can use an optional option.